apps/prairielearn/src/schemas/infoAssessment.ts

```diff
@@ -0,0 +1,406 @@
+import { z, type ZodSchema } from 'zod';
+
+import { CommentJsonSchema } from './comment.js';
+
+function uniqueArray<T extends ZodSchema>(schema: T) {
+  return z.array(schema).refine((items) => new Set(items).size === items.length, {
+    message: 'All items must be unique, no duplicate values allowed',
+  });
+}
```

Comment:
Is adding uniqueness checking safe? That is, could in break existing JSON files, or were we already validating this elsewhere?

Reply:
This was validated in the JSONSchema already previously with `uniqueItems: true`.

Reply:
Ah! Nice.